src/snowflake/snowpark/session.py

```diff
@@ -1026,6 +1044,7 @@ def write_pandas(
                 quote_identifiers=quote_identifiers,
                 auto_create_table=auto_create_table,
                 create_temp_table=create_temp_table,
+                overwrite=overwrite,
```

Comment:
remember to bump the connector dependency version to be >2.7.11 in which the new feature is released

Reply:
@sfc-gh-aling Thanks. Where/how do I do that?

Reply:
it's in setup.py: https://github.com/snowflakedb/snowpark-python/blob/main/setup.py#L13

Reply:
https://github.com/snowflakedb/snowpark-python/blob/196cb4e9d3fd4a862bf07f55c9c16433925f3b49/setup.py#L13 update it here

Reply:
Done. Thanks!